skycoin/skywire
# Secure VPN

Did you run `make format && make check`?
Yes
Fixes https://github.com/SkycoinPro/skywire-services/issues/312

 Changes:	
- Flag `-secure` was added to the VPN server. Default value is `true`. Used to turn on/off secure mode;
- In secure mode VPN server doesn't allow client/server SSH and doesn't allow any traffic from VPN clients to the server local network.

How to test this PR:
1. Run VPN server having some other machine in its local network;
2. Run VPN client on the machine connected to the separate network, establish VPN connection;
3. Try SSH'ing from VPN client machine to VPN server machine or any of machines in its local network. SSH should just hang.
